SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,150 @@
+
+*Image: The Unreal Engine Project Browser, with the project file path and project name highlighted.*
+
+After you have selected **Create Project**, the Unreal Engine generates the necessary project files and directories, it then closes the Editor and automatically opens Visual Studio.
+
+### Clone the GDK
+
+Now you need to clone the SpatialOS GDK for Unreal into your project. To do this: 
+
+1. In **File Explorer**, navigate to the `<Game>` directory and create a `Plugins` folder in this directory.
+2. In a Git Bash terminal window, navigate to `<Game>\Plugins` and clone the [GDK for Unreal](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+**Note:** You need to ensure that the root directory of the GDK for Unreal repository is called `UnrealGDK` so the file path is: `<Game>\Plugins\UnrealGDK\...`
+
+### Build the dependencies 
+
+To use the Starter Template, you must build the GDK for Unreal module dependencies and then add the GDK to your project. To do this: 
+
+1. Open **File Explorer**, navigate to the root directory of the GDK for Unreal repository (`<Game>\Plugins\UnrealGDK\...`), and double-click **`Setup.bat`**. If you haven't already signed into your SpatialOS account, the SpatialOS developer website may prompt you to sign in.
+1. In **File Explorer**, navigate to your <Game> directory and double-click **`<YourProject>`.sln** to open it with Visual Studio.
+1. In the Solution Explorer window, right-click on **`<YourProject>`** and select **Build**.
+1. Open **`<YourProject>`.uproject** in the Unreal Editor. 
+1. On the GDK for Unreal toolbar, select [**Schema**](https://docs.improbable.io/reference/latest/shared/glossary) to generate schema.<br/>
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select **Schema**_<br/>
+1. Select [**Snapshot**]({{urlRoot}}/content/generating-a-snapshot) to generate a snapshot.<br/>
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/snapshot-button.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select **Snapshot**_<br/>
+
+
+### Deploy your project 
+
+#### Deploy locally
+
+1. In the Unreal Editor, on the SpatialOS GDK toolbar, select **Start**. This opens a terminal window and starts a local SpatialOS deployment. Wait until you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector` in your terminal window.<br/>
+![Toolbar]({{assetRoot}}assets/set-up-template/template-start.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select **Start**_<br/>
+1. On the Unreal Editor toolbar, open the **Play** drop-down menu.
+1. Under **Multiplayer Options**, set the number of players to **2** and check the box next to **Run Dedicated Server**. Then, under **Modes**, select **New Editor Window (PIE)**.<br/>
```

Comment:
Run Dedicated Server is set to true by default for GDK projects. Perhaps just tell them to confirm it's set?

Also, should this heading be updated to *Deploy locally with multiple clients* ? As there's no obligation for them when testing locally to run two clients.

Reply:
Updated the heading per your suggestion. Thanks <3